docs/diagrams/src/Gateway.puml

```diff
@@ -13,35 +13,27 @@ LAYOUT_TOP_DOWN()
 
 Person(user, "End User")
 Component(arweave, "Arweave Blockweave")
+Component(network, "AR.IO Network")
 
 System_Boundary(ar-io, "AR.IO Gateway"){
     Component(core, "AR.IO Core Service")
     Component(envoy, "Envoy Service")
     Component(observer, "Observer Service")
 
-    System(storage, "Local Storage"){
-        ContainerDb(chunks, "Chunk data")
-        ContainerDb(headers, "Header Data")
-        ContainerDb(contiguous, "Contiguous Data")
-        ContainerDb(indexes, "Index Data")
-        ContainerDb(reports, "Observer Reports")
-    }
+    ContainerDb(index, "Chain and Bundle Index", "Chunked, header, and contiguous data and Observer reports")
```

Comment:
Oh, sorry, I wasn't specific enough in my comment. I was only trying to suggest that "Index Data" should be renamed to "Chain and Bundle Index" not that the entire group should be combined into one item.